### DISCUSSION TITLE
JS minification fails with PHP block comments

### DISCUSSION THREAD

**Discussion by luukes** [thuliteio/doks#706] [Q&A] [answered]
Hi,

I'm posting this issue here instead at gohugos page as I was not able to reproduce the error with a plain hugo project and would appreciate your help.

EDIT: this happens with flexsearch enabled - the minifcation error occurs while minifying index.js which contains the page content inside the flexsearch code.

### Steps to reproduce
Adding a code block (with triple backticks) that contains PHP code with a block comment fails to minimize the resulting `index.js` (flexsearch part).
```php
<?php
/*
 * Test
 */
public function doesNotMinify():int {
    return 42;
}
```

### Expected result
A successful build with JS minification. It works without minification.

### Actual result
The minification fails
```
Error: Error building site: MINIFY: failed to transform "index.js" (application/javascript): unexpected \ on line 277 and column 959
  277: ...hp\n/*\n * Test\n */\npublic function doe...
```

### Environment
```
> doks-child-theme@0.4.0 precheck
> npm version

{
  'doks-child-theme': '0.4.0',
  npm: '8.1.2',
  node: '16.13.2',
  v8: '9.4.146.24-node.14',
  uv: '1.42.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '93',
  nghttp2: '1.45.1',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '1.1.1l+quic',
  cldr: '39.0',
  icu: '69.1',
  tz: '2021a',
  unicode: '13.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}

> doks-child-theme@0.4.0 check
> exec-bin node_modules/.bin/hugo/hugo version

hugo v0.92.1-85E2E862+extended linux/amd64 BuildDate=2022-01-27T11:44:41Z VendorInfo=gohugoio
```

It works if I'm disabling only the JS minification in my configuration
```toml
[minify]
  disableJS = true
```

I've also tried to update to the latest hugo version 0.94.2 but it still failed.
Thanks

**Comment by h-enk** [comment, marked as answer]
See: https://github.com/h-enk/doks/issues/716
